spec/factories/course_assessment_answers.rb

```diff
@@ -3,9 +3,26 @@
     transient do
       course { build(:course) }
       assessment { build(:assessment, course: course) }
+      submission_traits []
     end
 
-    submission { build(:course_assessment_submission, assessment: assessment) }
+    submission { build(:course_assessment_submission, *submission_traits,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.